{"uuid": "b96e5910-ecdc-4e70-a1b4-9f2ba8ae0c82", "children": ["2aac2583-98de-48aa-853d-1d02917cf245", "097f58a0-9510-43c8-8a24-0cb7ae91be59", "490bf27f-99c8-4350-a8d4-c45a48b35161"], "befores": [{"name": "setup_profiles", "status": "passed", "attachments": [{"name": "ucentral_config: ", "source": "953bb94a-29e7-4d0a-866f-8555eec4f420-attachment.json", "type": "application/json"}, {"name": "/configure response: 200", "source": "88ef435f-f232-47b7-bc75-a87f569cc7cd-attachment.json", "type": "application/json"}, {"name": "Config Sent from API: ", "source": "99949316-ed03-4d09-b1f8-c247992f6475-attachment.json", "type": "application/json"}, {"name": "Latest Config Received by AP: ", "source": "c256ab69-9da5-40ae-b05c-2b14ef893062-attachment.json", "type": "application/json"}, {"name": "Time Took to apply Config: 39.97958445549011", "source": "407df52a-324b-42c3-be39-b28728cbed3b-attachment.attach"}, {"name": "Success : Active Config in AP: ", "source": "32624c6c-07d9-4af4-98fb-a08461d38fd6-attachment.attach"}, {"name": "iwinfo: ", "source": "22a9e786-f0b1-4384-92f4-6ca6aef35f49-attachment.attach"}, {"name": "AP Log: ", "source": "a91cc897-46f5-4280-af50-a0f70bcbc60f-attachment.attach"}, {"name": "wifi status", "source": "3d6a9f63-1536-4abf-bf37-82df5a802183-attachment.attach"}], "start": 1639114445617, "stop": 1639114613052}], "afters": [{"name": "setup_profiles::1", "status": "passed", "start": 1639115128689, "stop": 1639115128689}, {"name": "setup_profiles::teardown_session", "status": "broken", "statusDetails": {"message": "KeyboardInterrupt\n", "trace": "  File \"/home/imgd/TIP/venv/lib/python3.8/site-packages/allure_commons/_allure.py\", line 200, in __call__\n    return self._fixture_function(*args, **kwargs)\n  File \"/home/imgd/TIP/wlan-testing/tests/fixtures_2x.py\", line 718, in teardown_session\n    wifi_status = ap_ssh.get_wifi_status()\n  File \"/home/imgd/TIP/wlan-testing/tests/../libs/apnos/apnos.py\", line 424, in get_wifi_status\n    client = self.ssh_cli_connect()\n  File \"/home/imgd/TIP/wlan-testing/tests/../libs/apnos/apnos.py\", line 82, in ssh_cli_connect\n    client.connect(self.ip, username=self.username, password=self.password,\n  File \"/home/imgd/TIP/venv/lib/python3.8/site-packages/paramiko/client.py\", line 435, in connect\n    self._auth(\n  File \"/home/imgd/TIP/venv/lib/python3.8/site-packages/paramiko/client.py\", line 740, in _auth\n    self._transport.auth_publickey(username, key)\n  File \"/home/imgd/TIP/venv/lib/python3.8/site-packages/paramiko/transport.py\", line 1580, in auth_publickey\n    return self.auth_handler.wait_for_response(my_event)\n  File \"/home/imgd/TIP/venv/lib/python3.8/site-packages/paramiko/auth_handler.py\", line 231, in wait_for_response\n    event.wait(0.1)\n  File \"/usr/lib/python3.8/threading.py\", line 558, in wait\n    signaled = self._cond.wait(timeout)\n  File \"/usr/lib/python3.8/threading.py\", line 306, in wait\n    gotit = waiter.acquire(True, timeout)\n"}, "start": 1639115128689, "stop": 1639115130191}], "start": 1639114445617, "stop": 1639115130192}